Documentation/ceph-csi-drivers.md

```diff
@@ -202,14 +193,24 @@ mgr.a
 	caps: [mon] allow *
 	caps: [osd] allow *
 
-#encode admin/user key
-sh-4.2# echo -n AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==|base64
-QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
-#or
-sh-4.2# ceph auth get-key client.admin|base64
+#get  user key
+sh-4.2# ceph auth get-key client.admin
 QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
```

Comment:
(nit) To avoid confusion can you change the output to look like or as long as the output of the get-key without base64 encoding the same. (same in other places the example is used)

Reply:
yeah, forgot  to replace the key, updated now